api/client/service.proto

```diff
@@ -34,10 +36,62 @@ message ExecutionResponse {
   google.rpc.Status error_detail = 7;
 }
 
+// Used to request a RequestStreamResponse.
+message RequestStreamRequest {
+  // Indicates the amount of RequestStreamResponses that should be yielded by the server
+  // in response to this request.
+  // Used to implement basic flow control by the client. Once AsyncClientImpl gets a way
+  // to apply backpressure this could probably be dropped.
+  uint64 amount = 1;
```

Comment:
Nit: I would probably find `quantity` a bit clearer.

Reply:
Done in d43efd24a7d9991aca07b559dccd606c033f3322